package.json

```diff
@@ -113,6 +113,11 @@
           "default": "LL",
           "type": "string",
           "description": "Specifies the date format (using Moment.js syntax) that is used to for the title of daily pages."
+        },
+        "gistpad.tracing.enableOutputChannel": {
```

Comment:
Since VS Code already has the user-visible concept of an "Output" pane, I think we can just call this setting "gistpad.output". Introducing the concept of "tracing" is seems potentially unnecessary, since we just want to associate this setting with it enabling a GistPad stream in the Output pane.